README.md

```diff
@@ -21,6 +21,70 @@ Note: To be able to take advantage of alternate `define` method name, you will a
 build tooling generates using the alternate.  An example of this is done in the [emberjs-build](https://github.com/emberjs/emberjs-build)
 project in the [babel-enifed-module-formatter plugin](https://github.com/emberjs/emberjs-build/blob/v0.4.2/lib/utils/babel-enifed-module-formatter.js).
 
+## Public API
+
+`loader.js` provides the following as named exports from a `loader` module:
+
+```ts
+interface LoaderModule {
+  /*
+    Return the list of module id's that are present in the registry
+  */
+  getIds(): string[];
```

Comment:
this is the unique id, usually name is relative or not where as this is registry specific.

https://whatwg.github.io/loader/#loader-import  if you look here name is used for APIs like import() but the registry uses keys.

Would getKeys() be ok with you?

I hate naming things.